stable/chaoskube/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: chaoskube
-version: 0.7.1
-appVersion: 0.7.1
+version: 0.8.0
```

Comment:
Please keep this at `0.7.0`. It will makes sense later.

Reply:
👍 Done